perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -1259,8 +1257,8 @@ public Application createApplicationInternal(PerunSession session, Application a
 					if (loginAvailable) {
 						try {
 							// Reserve login
-							jdbc.update("insert into application_reserved_logins(login,namespace,app_id,created_by,created_at) values(?,?,?,?,?)",
-									login, loginNamespace, appId, application.getCreatedBy(), new Date());
+							jdbc.update("insert into application_reserved_logins(login,namespace,user_id,extsourcename,created_by,created_at) values(?,?,?,?,?)",
+									login, loginNamespace, userId, session.getPerunPrincipal().getExtSourceName(), session.getPerunPrincipal().getActor(), new Date());
```

Comment:
Related to this:
```
application.getCreatedBy() -> session.getPerunPrincipal().getActor()
```
we should just check how is original value constructed in WUI/registrar and if it can be changed.

Reply:
The principal's actor is used as createdBy in WUI, I just thought it would be safer to use the session's actor rather than what is sent from frontend to RPC (they should be the same, though).

Reply:
Ok, then its fine.